praw/models/reddit/wikipage.py

```diff
@@ -162,12 +380,43 @@ def mod(self) -> WikiPageModeration:
         """
         return WikiPageModeration(self)
 
+    @property
+    def revision_hidden(self) -> Union[bool, None]:
+        """Check the ``revision_hidden`` attribute of a WikiPage revision.
+
+        Example usage:
+
+        .. code-block:: python
+
+            one_revision = reddit.subreddit("test").wiki["praw"].revisions(limit=1)
+            wiki_page = next(one_revision).get("page")
+            wiki_page.revision_hidden
```

Comment:
```suggestion
            one_revision = next(reddit.subreddit("test").wiki["praw"].revisions(limit=1))
            wiki_page = one_revision.get("page")
            wiki_page.revision_hidden
```
Could we move the `next` to previous line?